Desafios/Estruturas Condicionais/Gabaritos/1-10.js

```diff
@@ -84,15 +84,15 @@ function robotFriend(mon, dad, me, brother, sister) {
 }
 
 /* 06 - Taxa Metabólica Basal */
-function basalMetabolicRate(age, sex, weight, heigh) {
+function basalMetabolicRate(age, sex, weight, height) {
   let bmr;
```

Comment:
metabolicRate
```suggestion
  let metabolicRate;
```

Faz sentido deixarmos essa variável mais nítida ?